plugins/schedule/base/lib/kScheduleEventsConsumer.php

```diff
@@ -5,20 +5,26 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
     {
         if ($object instanceof categoryEntry && $object->getStatus() == categoryEntryStatus::ACTIVE)
             return true;
+        if ($object instanceof ScheduleEventResource )
+            return true;
         return false;
     }
 
     public function shouldConsumeDeletedEvent(BaseObject $object)
     {
         if ($object instanceof categoryEntry)
             return true;
+        if ($object instanceof ScheduleEventResource )

```

Comment:
ScheduleEventResource is never deleted (changed to deleted status), it's erased (removed from the DB), it's a different event type.